src/test/java/com/commercetools/testUtil/ctpUtil/CleanupTableUtil.java

```diff
@@ -34,6 +38,20 @@ public static int cleanupPaymentTable(SphereClient sphereClient) {
         return removed;
     }
 
+    public static int cleanupOrders(SphereClient sphereClient) {
```

Comment:
looks like it's time to make a common internal method, which makes logs and cleanup, to avoid copy-paste code every time